python/cudf/cudf/core/window/rolling.py

```diff
@@ -256,17 +266,17 @@ def _apply_agg(self, agg_name):
         else:
             return self._apply_agg_dataframe(self.obj, agg_name)
 
-    def sum(self):
-        return self._apply_agg("sum")
-
-    def min(self):
-        return self._apply_agg("min")
-
-    def max(self):
-        return self._apply_agg("max")
+    def _reduce(
+        self, op: str, *args, **kwargs,
```

Comment:
Is this a `args`/`kwargs` we can also get rid of?

Reply:
No, `_reduce` needs the ability to parse arbitrary args and kwargs because other reductions might need to pass through additional parameters. For example, std and var have to pass through ddof. All of the actual reduction methods don't need to support arbitrary arguments, but `_reduce` needs to process whatever args those methods need to pass through the Cython->libcudf layers.